web/views.py

```
@@ -6949,3 +6951,168 @@ def delete_post(request, post_id):
+def infographics(request) -> HttpResponse:
+    """Display interesting facts about different subjects."""
+    subjects = Subject.objects.all().order_by("order", "name")
+
+    context = {
+        "subjects": subjects,
+    }
+
+    return render(request, "infographics/infographics.html", context)
+
+
+def api_get_subjects(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `api_get_subjects`

(ANN201)